docs/buildrun.md

```diff
@@ -80,6 +81,31 @@ spec:
     name: buildpack-nodejs-build-namespaced
 ```
 
+_Please note:_ The `BuildRef` and `BuildSpec` are mutually exclusive.
+
+### Defining the BuildSpec
```

Comment:
pls add this to the top index

Reply:
@HeavyWombat this still missing.

Reply:
@SaschaSchwarze0 @qu1queee Not sure why this is not marked as outdated here, but I moved the note to the common section: https://github.com/shipwright-io/build/pull/1016/commits/e735e50e5447bbe99243dbbbbd8540c795371076.

Reply:
@HeavyWombat I think I was not clear enough. You need to add the new heading level into the [index](https://github.com/shipwright-io/build/pull/1016/files#diff-a4578c2cbd97c42d96f1dc937794172a093bcd676f6d6f190b7452f54539996eR11-R14)

Reply:
Yeah, I misunderstood that. I am sorry. It should be fixed now.